Xamarin.Essentials/Maps/Map.android.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Globalization;
+using System.Threading.Tasks;
+using Android.Content;
+using AndroidUri = Android.Net.Uri;
+
+namespace Xamarin.Essentials
+{
+    public static partial class Map
+    {
+        internal static Task PlatformOpenMapsAsync(double latitude, double longitude, MapLaunchOptions options)
+        {
+            var uri = string.Empty;
+            if (string.IsNullOrWhiteSpace(options.Name))
+                uri = $"geo:{latitude.ToString(CultureInfo.InvariantCulture)},{longitude.ToString(CultureInfo.InvariantCulture)}?q={latitude.ToString(CultureInfo.InvariantCulture)},{longitude.ToString(CultureInfo.InvariantCulture)}";
+            else
+                uri = $"geo:{latitude.ToString(CultureInfo.InvariantCulture)},{longitude.ToString(CultureInfo.InvariantCulture)}?q={latitude.ToString(CultureInfo.InvariantCulture)},{longitude.ToString(CultureInfo.InvariantCulture)}({options.Name})";
+            StartIntent(uri);
+            return Task.CompletedTask;
+        }
+
+        internal static Task PlatformOpenMapsAsync(Placemark placemark, MapLaunchOptions options)
+        {
+            placemark = placemark.Escape();
+            var uri = $"http://maps.google.com/maps?q={placemark.Thoroughfare} {placemark.Locality}, {placemark.AdminArea} {placemark.CountryName}";
```

Comment:
Same here. this would be for google maps specifically, else we should use geo intent.

Reply:
My bad, changed it to geo uri